README.md

```diff
@@ -1,4 +1,6 @@
-# Metrilo custom integration
+# Metrilo custom integration (API v2)
+
+:warning: If you have integrated with the previous version of our API, the guide has been moved [here](api_v1.md). 
```

Comment:
No need for a warning. We're trying to focus our clients to the latest version and just give a chance for existing ones to figure out how to implement. Emphasise on the lates API. Do not scare people with warnings.